src/pages/Donate/DonateForm.vue

```diff
@@ -197,11 +188,11 @@ export default {
 					});
 				} else {
 					this.$kvTrackEvent(
```

Comment:
The add a one time donation to basket should be a single analytics event, no matter where this form is displayed or what the route it. Cleaned this up and removed `formSubmitAnalytics` prop